ipyleaflet/leaflet.py

```diff
@@ -644,8 +644,8 @@ class SplitMapControl(Control):
     _view_name = Unicode('LeafletSplitMapControlView').tag(sync=True)
     _model_name = Unicode('LeafletSplitMapControlModel').tag(sync=True)
 
-    left_layer = Instance(Layer).tag(sync=True, **widget_serialization)
-    right_layer = Instance(Layer).tag(sync=True, **widget_serialization)
+    left_layer = List(Instance(Layer)).tag(sync=True, **widget_serialization)
```

Comment:
For backward compatibility purpose, I suggest we use an `Union((Instance(Layer), List(Instance(Layer))))`